security/firely-auth/firely-auth-settings.rst

```diff
@@ -488,7 +488,7 @@ External identity providers
 			"DisplayName": "Login via SSO - <Name of IdentityProvider>",
 			"ClientId": "ClientId for Firely Auth, pre-registered with external service",
 			"ClientSecret": "secret for clientId",
-			"AllowAutoProvision": true|false,
```

Comment:
Using values like these will cause the whole JSON codeblock to fail at rendering + syntax highlighting. But we should use JSON since it's easier to read, instead of a generic code block.